src/test/java/com/synopsys/integration/alert/performance/utility/AuditCompleteWaitJobTask.java

```diff
@@ -154,4 +166,42 @@ private String createAuditRequestString(int pageNumber, int pageSize, Notificati
             URLEncoder.encode(String.format("\"%s\"", notificationType.name()), StandardCharsets.UTF_8)
         );
     }
+
+    private void logAverageAuditTime() throws IntegrationException {
+        List<AuditJobStatusModel> auditJobStatusModels = new ArrayList<>();
+        int pageNumber = 0;
+        AuditEntryPageModel auditEntryPageModel = getPageOfAuditEntries(pageNumber, 100);
+        do {
+            auditJobStatusModels.addAll(auditEntryPageModel.getContent().stream()
+                .map(AuditEntryModel::getJobs)
+                .flatMap(List::stream)
+                .map(JobAuditModel::getAuditJobStatusModel)
+                .collect(Collectors.toList()));
+            pageNumber++;
+            auditEntryPageModel = getPageOfAuditEntries(pageNumber, 100);
+        } while (auditEntryPageModel.getCurrentPage() < auditEntryPageModel.getTotalPages());
+
+        OptionalDouble averageAuditTime = auditJobStatusModels.stream()
+            .map(this::calculateAuditTimeDifference)
+            .mapToDouble(a -> a)
+            .average();
+
+        if (averageAuditTime.isEmpty()) {
+            intLogger.info("Performance: Could not calculate average audit time.");
+            return;
+        }
+        intLogger.info(String.format("Performance: Average audit time: %s seconds.", averageAuditTime.getAsDouble()));
+    }
+
+    private Long calculateAuditTimeDifference(AuditJobStatusModel auditJobStatusModel) {
+        try {
+            OffsetDateTime timeCreated = DateUtils.parseDate(auditJobStatusModel.getTimeAuditCreated(), DateUtils.AUDIT_DATE_FORMAT);
+            OffsetDateTime timeLastSent = DateUtils.parseDate(auditJobStatusModel.getTimeLastSent(), DateUtils.AUDIT_DATE_FORMAT);
+            return ChronoUnit.SECONDS.between(timeCreated, timeLastSent);
```

Comment:
Can you change the method to return a double and use the doubleValue() method of a Long object.

Reply:
Thought of this after the fact